src/store/Chat/index.js

```diff
@@ -31,7 +31,8 @@ const SEND_MESSAGE = 'CHAT_MESSAGE_OPERATOR';
 
 const CLIENT_TYPING = 'CHAT_CLIENT_TYPING';
 const CLIENT_IDLE = 'CHAT_CLIENT_IDLE';
-const RECEIVE_MESSAGE = 'CHAT_MESSAGE_CLIENT';
+export const RECEIVE_MESSAGE = 'CHAT_MESSAGE_CLIENT';
```

Comment:
`const RECEIVE_MESSAGE = 'CHAT_RECEIVE_MESSAGE'` would be more consistent with the const name.

Reply:
i agree

Reply:
but i guess it doesn't really matter since the strings are just supposed to be arbitrary unique strings so that constants can minimize the chance of misspelling an action type.